CHANGELOG.md

```diff
@@ -4,7 +4,16 @@ All notable changes to the t2iapi module will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
 
+### Update
+
+- update used PYTHON_VERSION: 3.8
+
+### Removed
```

Comment:
If you remove / change manipulations then you have to update the BASE_PACKAGE_VERSION too.
BASE_PACKAGE_VERSION should be 2.0.0

Reply:
done

Reply:
as discussed it is not necessary to list the version updates here